src/da65/scanner.c

```diff
@@ -484,15 +655,19 @@ void InfoSetName (const char* Name)
 /* Set a name for a config file */
 {
     InfoFile = Name;
+    xfree(InputSrcName);
+    InputSrcName = xstrdup(Name);
 }
 
 
 
+#ifdef unused
```

Comment:
I very strongly dislike commented-out code without a comment giving the reason for commenting out the code. So please either remove or elaborate.

Reply:
The InfoGetName() has been unused at least since 2.13.3. Maybe it was implemented for future extensions or is just a dead code.

Reply:
Thanks for the feedback. If InfoGetName() is never called since quite some time I ask you to completely remove it.